ui/raidboss/data/06-ew/raid/p9s.ts

```diff
@@ -503,16 +507,24 @@ const triggerSet: TriggerSet<Data> = {
       type: 'Ability',
       netRegex: { id: '8180', source: 'Kokytos', capture: false },
       condition: (data) => data.limitCutDash > 0 && data.limitCutDash < 4,
-      alertText: (data, _matches, output) => {
+      preRun: (data) => {
         const activePlayers = limitCutPlayerActive[data.limitCutDash];
         if (activePlayers === undefined)
           return;
-        const [dashPlayer, soakPlayer] = activePlayers;
-        if (dashPlayer === undefined || soakPlayer === undefined)
+        [data.lc1DashPlayer, data.lc1SoakPlayer] = activePlayers;
+      },
+      delaySeconds: (data) => {
+        // delay 'soak tower' call by 1 second to prevent confusion due to ability timing
+        if (data.lc1SoakPlayer !== undefined && data.limitCutNumber === data.lc1SoakPlayer)
+          return 1;
+        return 0;
```

Comment:
```suggestion
        return limitCutPlayerActive[data.limitCutDash][1] === data.limitCutNumber ? 1 : 0,
```

Sorry, just looking at this and wondering why we need an extra variable or two on `Data`.  It seems like there's not a need to store anything extra, or everything is already there? Would something like this be sufficient to solve this problem?

Reply:
> Sorry, just looking at this and wondering why we need an extra variable or two on `Data`. It seems like there's not a need to store anything extra, or everything is already there? Would something like this be sufficient to solve this problem?

I added the additional vars on data to allow preRun to figure out who was getting dashed/soaking, so that could then be used in delaySeconds, and the logic in alertText was then simplified.  But this approach works too; it would just mean moving the preRun logic back down to alertText.